deploy/weekly-environment/variables.tf

```diff
@@ -7,7 +7,7 @@ variable "ec_api_key" {
 
 variable "ess_region" {
   default     = "gcp-us-central1"
-  description = "Optional ESS region where the deployment will be created. Defaults to gcp-us-west2"
+  description = "Optional ESS region where the deployment will be created. Defaults to gcp-us-central1"
```

Comment:
why did you change the region?

Reply:
I didn't change the region, I just change the description so it will be aligned with the default value.

Reply:
So I think it should be aligned with the rest of the repo to `gcp-us-west2` as this is the region they publish the BCs to AFAIK

Reply:
But we use staging here, and staging contains only one region for GCP - `us-centeral1`